README.md

```diff
@@ -1 +1,50 @@
-# Bubble-Sort
\ No newline at end of file
+# Bubble Sort
+
+This is a collaborative project about building a bubble sort algorithm.
+
+![Bubble Sort Animation](Bubble-sort-example.gif)
```

Comment:
This animation is not rendering. It seems not included in the repo or the link is invalid.